index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <h1 class="logo">John Doe</h1>
+       <div class="harmburger flex">
+        <span class="line"></span>
+        <span class="line"></span>
+        <span class="line"></span>
+       </div>
+      </nav>
+      <section class="headline flex">
+        <h1>Hey There.
+            I’m David
+        </h1>
+        <h3>I’am a Software Developer</h3>
+        <div class="headline-details">
+            <p class="lead">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <div class="icons flex">
+                <img class="github" src="asset/github.png" alt="github"/>
+                <span class="in flex">
+                    <span class="i flex">
+                        <img src="asset/dot.png" alt="dot">
+                        <img src="asset/i.png" alt="i">
+                    </span>
+                    <img class="in" src="asset/magnet.png" alt="n"/>
+                </span>
+                <img class="rabbid" src="asset/rabbid.png" alt="rabbid"/>
+                <img class="twitter" src="asset/twitter.png" alt="twitter"/>
+                <img class="m" src="asset/m.png" alt="github"/>

```

Comment:
- [ ] The image list should be wrapped in a `ul` tag and each image on the `li` tag with an `a` tag, so the images work like links and are organized in unorder list